client/src/pages/user/user.jsx

```diff
@@ -0,0 +1,54 @@
+import s from './user.module.sass';
+import Avatar from '../../components/avatar';
+import {useNavigate} from "react-router-dom";
+
+const User = ({}) =>{
+
+    const navigate = useNavigate();
+
+    return(
+        <>
+            <div className={s.userWrapper}>
+
+                <div className={s.userHeader}>
+                    <button onClick={()=>navigate('/chat')}>
+                        <span>Cancel</span>
+                    </button>
+
+                    <div className={s.userAvatar}>
+                        <Avatar url={'https://avochka.ru/img/kartinka/1/enot_glass.jpg'}
+                                style={{width: '100px', height: '100px'}}
```

Comment:
stuff like this should be done by proper classes

Reply:
I corrected